### PR TITLE
Implement postgresIsTransacting()

### LINKED ISSUE #349: Export connection_is_transacting
It would be very practical if `connection_is_transacting` was exported in some way. Unfortunately it is not part of any functionality defined in DBI but it could maybe included in dbGetInfo? 

If you agree I can open a PR

I want to avoid using unexported functions like that:

```r
RPostgres:::connection_is_transacting(db@ptr)
```

### PR DESCRIPTION
**Pull request by jakob-r**
Closes #349

### CODE REVIEW

**Comment by krlmlr** [comment]
Thanks. Can you please document the new function in `_pkgdown.yml` and fix the failing test (`expect_fale`) ?

File: R/PqConnection.R [hunk 1]
```diff
@@ -308,6 +308,19 @@ postgresWaitForNotify <- function (conn, timeout = 1) {
...
+#'
+#' Detect whether the transaction is active for the given connection. A
+#' transaction might be started with [dbBegin()] or wrapped within
+#' [DBI::dbWithTransaction()].
+#' @export
+#' @param conn a [PqConnection-class] object, produced by
+#'   [DBI::dbConnect()]
+#' @return A boolean, indicating if a transaction is ongoing.
+postgresIsTransacting <- function(conn) {
+  connection_is_transacting(conn)
```
Comment (galachad) [inline R/PqConnection.R:321]: This should be a pointer.
```suggestion
  connection_is_transacting(conn@ptr)
```

File: tests/testthat/test-isTransacting.R [hunk 2]
```diff
@@ -0,0 +1,11 @@
+context("isTransacting")
+
+test_that("isTransacting detects transactions correctly", {
+  skip_on_cran()
+  db <- postgresDefault()
+  expect_fale(postgresIsTransacting(con))
```
Comment (galachad) [inline tests/testthat/test-isTransacting.R:6]: Typo
```suggestion
  expect_false(postgresIsTransacting(con))
```

File: tests/testthat/test-isTransacting.R [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  dbBegin(con)
+  expect_true(postgresIsTransacting(con))
+  dbCommit(con)
+  expect_fale(postgresIsTransacting(con))
```
Comment (galachad) [inline tests/testthat/test-isTransacting.R:10]: Typo
```suggestion
  expect_false(postgresIsTransacting(con))
```

File: tests/testthat/test-isTransacting.R [hunk 4]
```diff
@@ -0,0 +1,11 @@
+context("isTransacting")
+
+test_that("isTransacting detects transactions correctly", {
+  skip_on_cran()
+  db <- postgresDefault()
```
Comment (krlmlr) [inline tests/testthat/test-isTransacting.R:5]:
```suggestion
  con <- postgresDefault()
```

**Comment by krlmlr** [comment]
Thanks!

**Comment by krlmlr** [comment]
One more thing: do you agree to contribute under the MIT license? Please indicate your consent or disapproval in #312. Thanks!

**Comment by jakob-r** [comment]
Sorry for the lousy PR and thanks for fixing it :)

**Pull request merged**